scene/3d/skeleton_3d.cpp

```diff
@@ -95,6 +95,20 @@ bool Skeleton3D::_set(const StringName &p_path, const Variant &p_value) {
 		set_bone_pose_rotation(which, p_value);
 	} else if (what == "scale") {
 		set_bone_pose_scale(which, p_value);
+#ifndef DISABLE_DEPRECATED
+	} else if (what == "pose") {
+		// Kept for compatibility from 3.x to 4.x.
+		WARN_DEPRECATED_MSG(vformat(
+				"Skeleton uses old pose format, which is deprecated (and loads slower). Consider re-importing or re-saving the scene. Path: \"%s\"",
```

Comment:
I'd suggest making the path part optional, otherwise it just prints an empty and confusing path, something like:
```cpp
		WARN_DEPRECATED_MSG(
				"Skeleton uses old pose format, which is deprecated (and loads slower). Consider re-importing or re-saving the scene." + 
				is_inside_tree() ? vformat(Path: \"%s\"", get_path()) : String());
```

Reply:
Makes sense, could you amend the commit with this tiny edit @nikitalita?

Reply:
Done.